gemrb/GUIScripts/InventoryCommon.py

```diff
@@ -253,6 +272,97 @@ def GetItemDescription (item, itemtype):
 	text = searchRE.sub(replacement, text)
 	return text
 
+def GetIWD2RestrictedAlignments(mask):
+	text = ""
+	if mask & 0x8000: # neutral
+		text += GemRB.GetString(31535) + "\n  "
+	if mask & 0x4000: # good
+		text += GemRB.GetString(31532) + "\n  "
+	if mask & 0x2000: # evil
+		text += GemRB.GetString(31537) + "\n  "
+
+	return text
+
+def GetIWD2RestrictedStances(mask):
+	text = ""
+	if mask & 0x20000: # neutral
+		text += GemRB.GetString(31535) + "\n  "
+	if mask & 0x10000: # lawful
+		text += GemRB.GetString(29372) + "\n  "
+	if mask & 0x1000: # chaotic
+		text += GemRB.GetString(31536) + "\n  "
+
+	return text
+
+def GetIWD2RestrictedRaces(mask):
+	text = ""
+	n = CommonTables.Races.GetRowCount()
+
+	for i in range(n):
+		rowName = CommonTables.Races.GetRowName(i)
+		raceMask = CommonTables.Races.GetValue(rowName, "USABILITY")
+		nameRef = CommonTables.Races.GetValue(rowName, "NAME_REF")
+
+		if raceMask == 0x7FFFFFFF: # GetValue gives as signed value
+			raceMask = 0x80000000
```

Comment:
A little out of ideas how to better handle this for now.

Reply:
Nothing much better comes to mind.